test/subspace-test-runtime/src/lib.rs

```diff
@@ -1175,6 +1154,19 @@ impl_runtime_apis! {
             extract_successful_bundles(domain_id, extrinsics)
         }
 
+        #[allow(clippy::type_complexity)]
```

Comment:
If we hinting clippy for the same type at so many places then this types needs to aliased instead.